cmd/hw002/fibo/fibo.go

```diff
@@ -0,0 +1,39 @@
+package fibo
+
+import (
+	"errors"
+	"fmt"
+	"log"
+)
+
+var (
+	errWrongArg      = "error: wrong arg passed"
+	err64bitOverflow = "error: 64 bit overflowed : rtn size maximum equals 93"
+)
+
+// SizedSequence64bit returns Fibonacci array of required length
+func SizedSequence64bit(size int64) ([]int64, error) {
+	if size <= 0 {
+		_ = log.Output(1, "fibo: wrong input")
+		return nil, errors.New(errWrongArg)
+	}
+	var (
+		counted int64
+		a       int64
+		b       int64 = 1
+		rtn           = []int64{a, b} // init with first two
+	)
+	for i := int64(2); i <= size; i++ {
```

Comment:
As you know the size of the sequence you can set the slice capacity (to avoid extending of underlying array)
```suggestion
		rtn           = make([]int64, 0, size) 
	)
	rtn = append(rtn, a, b) // init with first two
	
	for i := int64(2); i <= size; i++ {
```

Reply:
fixed